build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/views/service_module.rb

```diff
@@ -51,6 +51,12 @@ def requires
         @service.gem_dependencies.keys
       end
 
+      # @return [Boolean] - Return true if a check is needed before
+      # requiring core to prevent circular dependencies.
+      def require_core_guard?
```

Comment:
i'd change this comment to mention backwards compatibility (SSO moved into core)